src/org/javarosa/core/model/utils/DateUtils.java

```diff
@@ -69,6 +68,23 @@ public DateFields () {
 //            tzOffset = 0;
         }
 
+        public DateFields(int year, int month, int day, int hour, int minute, int second, int secTicks) {
+            this.year = year;
+            this.month = month;
+            this.day = day;
+            this.hour = hour;
+            this.minute = minute;
+            this.second = second;
+            this.secTicks = secTicks;
+            // The official API returns a day of week [1=monday, 7=sunday].
+            // We normalize to current JR standard [0=sunday, 6 saturday] by getting the module of 6
```

Comment:
Not “module”, “modulo”.

% is the modulo operator and 7 is the modulus
normalized = ISO-8601 day of week modulo 7

Reply:
Thanks Dave! In Spanish is "módulo" for module, modulo, and modulus :D